frontend/public/components/factory/table.tsx

```diff
@@ -792,7 +792,7 @@ export const Table = connect<
               role={virtualize ? 'presentation' : 'grid'}
```

Comment:
Because of this presentation we should force rowgroup for the header.

See also https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role